src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
If we can get the author of pecl-event to fix the PHP 5.x variant of the extension to act just like the PHP 7.x one, then the 3rd option is definitely the best: recent versions of pecl-event are still perfectly compatible with older releases of PHP down to 5.4 and there shouldn't be any breaking change between 1.x and 2.x affecting our event loop implementation, so we could encourage users to upgrade to the latest verson of the extension by having a note in both the readme and the changelog.

In the meanwhile I can open an issue on [pecl-event](https://bitbucket.org/osmanov/pecl-event/issues?status=new&status=open) to let the author know about this.